Library/src/main/java/dev/testify/ScreenshotRule.kt

```diff
@@ -587,9 +570,9 @@ open class ScreenshotRule<T : Activity> @JvmOverloads constructor(
                     return
                 }
 
-                beforeInitializeView(activity)
+                screenshotLifecycleObservers.map { it.beforeInitializeView(activity) }
```

Comment:
Can use `forEach` instead since we don't care about the result.

Reply:
Yes, good call.